src/Symfony/Component/Yaml/Inline.php

```diff
@@ -562,6 +562,8 @@ private static function evaluateScalar($scalar, $flags, $references = array())
                         }
 
                         return;
+                    case 0 === strpos($scalar, '!php/const:'):
+                        return @constant(substr($scalar, 11));
```

Comment:
Hm, on the other hand they may have to deal with YAML files that did not write themself. But that's rather and argument to completely turn off parsing constants by default. Thus having the developer deliberately turn it on when wanting to parse constants.